StorageUtility/Linux/StorageUtility.h

```diff
@@ -0,0 +1,77 @@
+/*
+<License>
+Copyright 2015 Virtium Technology
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http ://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+</License>
+*/
+#ifndef __StorageUtility_h__
+#define __StorageUtility_h__
+
+#include <vector>
+
+#include "ErrorCodes.h"
+#include "BasicTypes.h"
+#include "vtStorPlatformDefines.h"
+
+#include "DeviceInterface.h"
+
+#include <libudev.h>

```

Comment:
Please move the #include <libudev.h> right after the #include <vector>

The inclusion is same inclusion of code of master: not because of other make it wrong then you leave it behind
We're developer, so if we see some mistakes of others, we could fix it for them please remember